.env.example

```
@@ -17,11 +17,11 @@ DISCORD_APPLICATION_PUBLIC_KEY=""
 DISCORD_APPLICATION_BOT_TOKEN=""
 
 # Discord Default Activity Status
-DISCORD_DEFAULT_STATUS=""Spreading Paw-sitivity 🐾"
+DISCORD_DEFAULT_STATUS="Spreading Paw-sitivity 🐾"
 # DISCORD_DEFAULT_ACTIVITY_TYPE can be "Playing", "Streaming", "Listening", "Watching", or "Custom"
 DISCORD_DEFAULT_ACTIVITY_TYPE="Custom"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)